docs/v8-new-performance-apis.md

```diff
@@ -0,0 +1,250 @@
+the decision to align the performance APIs for all JavaScript-based SDKs.
+
+## The "old" Way of Manual Performance Instrumentation
+
+Previously, there where two key APIs for adding manual performance instrumentation to your applications:
+
+- `startTransaction()`
+- `span.startChild()`
+
+This showed the underlying data model that Sentry was originally based on, which is that there is a root **Transaction**
+which can have a nested tree of **Spans**.
+
+## The new model: Goodbye Transactions, Hello Spans Everywhere!
+
+In the new model, transactions are conceptually gone. Instead, you will _always_ operate on spans, no matter where in
+the tree you are. Note that in the background, spans _may_ still be bundled up to transaction for the UI. However, this
+happens transparently, and from an SDK perspective, all you have to think about are spans.
+
+## The Span schema
+
+Previously, spans & transactions had a bunch of properties and methods to be used. Most of these have been removed in
+favor of a slimmer, more straightforward API, which is also aligned with OpenTelemetry Spans. You can refer to the table
+below to see which things used to exist, and how they can/should be mapped going forward:
+
+| Old name              | Replace with                                         |
+| --------------------- | ---------------------------------------------------- |
+| `traceId`             | `spanContext().traceId`                              |
+| `spanId`              | `spanContext().spanId`                               |
+| `parentSpanId`        | Same                                                 |
+| `status`              | TODO: new signature                                  |
+| `sampled`             | `spanContext().traceFlags`                           |
+| `startTimestamp`      | `startTime` - note that this has a different format! |
+| `tags`                | Use `attributes` instead, or set tags on the scope   |
+| `data`                | Use `attributes` instead.                            |
+| `transaction`         | Removed                                              |
+| `instrumenter`        | Removed                                              |
+| `finish()`            | `end()`                                              |
+| `end()`               | Same                                                 |
+| `setTag()`            | `setAttribute()`                                     |
+| `setData()`           | `setAttribute()`                                     |
+| `setStatus()`         | TODO: new signature                                  |
+| `setHttpStatus()`     | ???                                                  |
+| `setName()`           | TODO: `updateName()`                                 |
+| `startChild()`        | Call `Sentry.startSpan()` independently              |
+| `isSuccess()`         | ???                                                  |
```

Comment:
`isSuccess()` is not used by us (and really only checks the status), so I think we can deprecate/remove this.